docs/content/usage/cli/Options.md

```diff
@@ -34,11 +34,11 @@ To run the CLI without `sudo`, you have four options:
 
 ## Port Usage
 
-By default lego assumes it is able to bind to ports 80 and 443 to solve challenges.
+By default, lego assumes it is able to bind to ports 80 and 443 to solve challenges.
```

Comment:
Do we have a canonical way to refer to this software? I've seen "lego", "Lego", and "LEGO".

This PR would be a great opportunity for make that consistent :)

Reply:
for me, it's "lego" or "LeGo" because it's not an acronym ("LEGO"), and "Lego" is a brick brand.

Reply:
Wouldn't "Let's Encrypt in Go" lend itself to the backcronym "LEGo"? "Le go" is certainly funny :)

Personally, I prefer "Lego", although that danish toy producer might object...

> I'm not sure to want to handle this topic now :smile: 

Completely understandable.